src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceContext.java

```diff
@@ -27,10 +27,8 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.model.TaskListener;
 
-import java.util.Collections;
-import java.util.EnumSet;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
Oh my bad, I need to add the `CONTRIBUTING.md` to this repo. See https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md for the content

(TL;DR no `*` imports please

Reply:
I thought I spotted all of them - thanks for the link, the intellij configuration tips should help.

Do you know what I should do about the hash code findbugs complaint? This class doesn't have any fields. Should I also mark `hashCode()` as abstract?